knotx-example/knotx-mocks/src/main/resources/content/local/noBody.html

```diff
@@ -0,0 +1,67 @@
+<!--
```

Comment:
Duplicate. We could try to configure repository to read templates from `/test/resources`.